cmd/core/pfcp_session_handlers.go

```diff
@@ -480,6 +481,16 @@ func composeFarInfo(far *ie.IE, localIp net.IP, farInfo ebpf.FarInfo) (ebpf.FarI
 				return ebpf.FarInfo{}, fmt.Errorf("IPv6 not supported yet")
 			}
 		}
+		forwardingPolicyIndex := findIEindex(forward, 41) // IE Type Outer Header Creation
```

Comment:
Copy-past typo in comment

Reply:
Thanks for picking this up, I have updated the comment.